scalability_and_performance/telco_ran_du_ref_design_specs/telco-ran-du-rds.adoc

```diff
@@ -0,0 +1,161 @@
+
+include::modules/telco-ran-machine-configuration.adoc[leveloffset=+3]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../edge_computing/ztp-reference-cluster-configuration-for-vdu.adoc#ztp-sno-install-time-cluster-config[Recommended {sno} cluster configuration for vDU application workloads].
+
+[id="telco-ran-du-deployment-components"]
+== Telco RAN DU deployment components
+
+The following sections describe the various {product-title} components and configurations that you use to configure the hub cluster with {rh-rhacm}.
+
+include::modules/telco-ran-red-hat-advanced-cluster-management-rhacm.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../edge_computing/ztp-deploying-far-edge-clusters-at-scale.adoc#about-ztp_ztp-deploying-far-edge-clusters-at-scale[Using {ztp} to provision clusters at the network far edge]
+
+* link:https://docs.redhat.com/en/documentation/red_hat_advanced_cluster_management_for_kubernetes[Red Hat Advanced Cluster Management for Kubernetes]
+
+include::modules/telco-ran-topology-aware-lifecycle-manager-talm.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../edge_computing/cnf-talm-for-cluster-upgrades.adoc#cnf-talm-for-cluster-updates[Updating managed clusters with the {cgu-operator-full}]
+
+include::modules/telco-ran-gitops-operator-and-ztp-plugins.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../edge_computing/ztp-preparing-the-hub-cluster.adoc#ztp-preparing-the-ztp-git-repository-ver-ind_ztp-preparing-the-hub-cluster[Preparing the {ztp} site configuration repository for version independence]
+
+* xref:../../edge_computing/policygentemplate_for_ztp/ztp-advanced-policy-config.adoc#ztp-adding-new-content-to-gitops-ztp_ztp-advanced-policy-config[Adding custom content to the {ztp} pipeline]
+
+include::modules/telco-ran-agent-based-installer-abi.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../installing/installing_with_agent_based_installer/installing-with-agent-based-installer.adoc#installing-with-agent-based-installer[Installing an {product-title} cluster with the Agent-based Installer]
+
+[id="telco-ran-du-reference-configuration-crs"]
+== Telco RAN DU reference configuration CRs
+
+Use the following custom resources (CRs) to configure and deploy {product-title} clusters with the telco RAN DU profile.
+Use the CRs to form the common baseline used in all the specific use models unless otherwise indicated.
+
+[NOTE]
+====
+You can extract the complete set of RAN DU CRs from the `ztp-site-generate` container image.
+See xref:../../edge_computing/ztp-preparing-the-hub-cluster.adoc#ztp-preparing-the-ztp-git-repository_ztp-preparing-the-hub-cluster[Preparing the {ztp} site configuration repository] for more information.
+====
+
+include::modules/using-cluster-compare-telco-ref.adoc[leveloffset=+1]
```

Comment:
This include ends up bringing the cluster compare procedure into the reference configuration CRs section, but it is out of context there and the procedure contains only a single step of "login to the container registry". Was this intended to be some other link, or a more expansive procedure describing how to compare against the reference CRs listed in the tables?

Reply:
asciidoc typo. fixed in next push